fraction.rb

```diff
@@ -1,64 +1,51 @@
 class Fraction
   def initialize(numerator, denominator)
-    if ! numerator.is_a?(Integer) && denominator.is_a?(Integer)
-      # TODO: Proper execption for this issue
-      # TODO: Handle case when one or both parameters are not int but could be easily transformed to int
-      raise 'these are supposed to be numbers: '+numerator.inspect+' '+ denominator.inspect
-    end
     @numerator = numerator
     @denominator = denominator
   end
-
```

Comment:
Why did you delete all these lines? It's a mess now. You should keep 1 line between methods descriptions. Read more about good style [here](https://github.com/bbatsov/ruby-style-guide)